src/ozone/ozone.py

```diff
@@ -420,21 +430,16 @@ def get_specific_parameter(
         Returns:
             float: Value of the specified parameter for the given city.
         """
-        result: float = 0.0
-        try:
-            r = self._make_api_request(
-                f"{self._search_aqi_url}/{city}/?token={self.token}"
-            )
-            if self._check_status_code(r):
-                data_obj = json.loads(r.content)["data"]
-                row = self._parse_data(data_obj, city, [air_param])[0]
-                result = float(row[air_param])
+        row = self.get_city_air(city, params=[air_param])
 
+        try:
+            result = float(row[air_param])
         except KeyError:
             print(
                 "Missing air quality parameter!\n"
                 + "Try: get_specific_parameter(`city name`, `aqi` or `no2` or `co`)"
             )
+            raise
```

Comment:
I tested this block of code and i noticed that the exception message that is raised here is still quite lengthy and not very descriptive and the reason for that is because we are raising the exception again at line (442) but this time (after removing the redundancy)  the exception comes from pandas, since `row` variable which is returned by `get_city_air(..)` is now a pandas data frame as opposed to being a dictionary object returned by `_parse_data()` before the changes 

My suggestion is the following: 
instead of throwing the same exception we catch we can simply raise a simple new exception like the following:

```suggestion
        try:
            result = float(row[air_param])
        except KeyError:
        	raise Exception("Missing air quality parameter!\n" +
                             "Try: get_specific_parameter(`city name`, `aqi` or `no2` or `co`)")

        return result
            
```

Reply:
As for the rest of the changes they work flawlessly :)

Reply:
Thanks for the perspective, I will change it.

Reply:
@Sam-damn great suggestion! I'll add it in directly from here.

Reply:
@lahdjirayhan , I didn't see your comment here and added this commit in directly already. Sorry about that!